fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/directory/AgilityContextTest.java

```diff
@@ -138,8 +150,9 @@ void testAgilityContextOneLongWrite(int loopCount, int sizeLimit, int timeLimit)
                         "And looked down one as far as I could\n" +
                         "To where it bent in the undergrowth;" ;
 
-        try (FDBRecordContext context = openContext(insertProps)) {
-            final AgilityContext agilityContext = AgilityContext.factory(context, true);
+        try (FDBRecordContext context = useProp ? openContext(insertProps) : openContext()) {
+            final AgilityContext agilityContext =
+                    useProp ? AgilityContext.agile(context) : AgilityContext.agile(context, timeLimit, sizeLimit);
```

Comment:
✅ Used timer. 
The time limit is now down to 1 millisecond, which appears to be constantly exceeded. I am not sure if I should hope that it will never change, or hope that fdb will become fast enough to challenge this limit.